### ISSUE TITLE
[Site] create Color page

### ISSUE THREAD

**Issue by riley** [mineral-ui/mineral-ui#239]
As a designer, I want to be able to understand what colors were chosen for Mineral, and in what contexts they should be used.

AC
- Card containing each of the defined color ramps with black/white text overlaid to show a11y
- copy outlining usage guidelines
- links to the Tokens page along with some inline examples
